src/Count/Count.tsx

```diff
@@ -4,6 +4,35 @@ import * as PropTypes from "prop-types";
 
 import Number from "../Number";
 
+interface BaseProps {
+  plural?: string;
+  short?: boolean;
+  singular?: string;
+  zeroOverride?: string;
+}
+
+interface FormatOptions extends BaseProps {
+  number: string | number;
+}
+
+export interface Props extends BaseProps {
+  children?: FormatOptions["number"];
+  className?: string;
+}
+
+const propTypes = {
+  children: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  className: PropTypes.string,
+  plural: PropTypes.string,
+  singular: PropTypes.string.isRequired,
+  short: PropTypes.bool,
+  zeroOverride: PropTypes.string,
+};
+
+export const cssClass = {
```

Comment:
I see that previously `cssClass` was loaded onto `Count` and as a result accessible via `Count.cssClass`. Though I highly doubt that any consumer is accessing `Count.cssClass`, we may wanna keep it as a static member to minimize breaking changes. I see that we've left the `static cssClass` on other components

Reply:
I searched sd2 and launchpad and didn't find any references, so even though it's a breaking change it shouldn't affect any consumers. Even though it's not being referenced anywhere, it feels like a bad pattern. I actually wanted to remove `cssClass` as a static property from all components, but I only ended up doing it for the ones that were converted from JavaScript. If we're going to make breaking changes, nows a good time.

As a side note, the reason I made this an export is so that the unit tests can still reference them. The unit tests use css selectors to find rendered elements. It would not be exported at the package level though.

If I can verify that these are not referenced by any consumers, would it be ok to remove these `cssClasses`, or should that be left for a future change?

Reply:
Ah yeah! If you can verify that it's not accessed, def feel free to remove it!

Reply:
I checked again and didn't find any external references to these, so I went ahead and removed them from all.